Buildings/Experimental/DHC/Networks/Steam/Examples/DistributionCondensatePipe.mo

```diff
@@ -0,0 +1,105 @@
+within Buildings.Experimental.DHC.Networks.Steam.Examples;
+model DistributionCondensatePipe
+  "Example model for the steam heating distribution network"
+  extends Modelica.Icons.Example;
+
+  package MediumSte = Buildings.Media.Steam "Steam vapor medium";
+  package MediumWat =
+    Buildings.Media.Specialized.Water.TemperatureDependentDensity
+    "Liquid water medium";
+
+  parameter Modelica.Units.SI.AbsolutePressure pSat=150000
+    "Saturation pressure";
+  parameter Modelica.Units.SI.Temperature TSat=
+     MediumSte.saturationTemperature(pSat)
+     "Saturation temperature";
+  parameter Modelica.Units.SI.SpecificEnthalpy dh_nominal=
+    MediumSte.specificEnthalpy(MediumSte.setState_pTX(
+        p=pSat,
+        T=TSat,
+        X=MediumSte.X_default)) -
+      MediumWat.specificEnthalpy(MediumWat.setState_pTX(
+        p=pSat,
+        T=TSat,
+        X=MediumWat.X_default))
+    "Standard change in enthalpy due to vaporization";
```

Comment:
Parameter description can be changed to "Nominal change in enthalpy due to vaporization" as used in `Buildings.Experimental.DHC.Networks.Steam.Examples.ConnectionCondensatePipe`

Reply:
Revised as suggested! Thanks for catching this.